File: baekjoon/12000/12606.py
T = int(input())
for t in range(1, T + 1):
    s = input().split()
    print("Case #" + str(t) + ": ", end='')
    for i in range(len(s) - 1, -1, -1):
        print(s[i], end=' ')
    print()

File: baekjoon/12000/12606.rs
use std::io::*;

fn main() {
    let mut s = String::new();
    stdin().read_to_string(&mut s).ok();
    let mut it = s.split('\n');
    let t = it.next().unwrap().parse::<usize>().unwrap();
    for t in 1..=t {
        print!("Case #{t}:");
        for word in it.next().unwrap().split_ascii_whitespace().rev() {
            print!(" {word}");
        }
        println!();
    }
}
